apps/website-new/docs/en/guide/troubleshooting/runtime/RUNTIME-001.mdx

```diff
@@ -0,0 +1,21 @@
+import ErrorCodeTitle from '@components/ErrorCodeTitle';
+import Table from '@components/Table'
+
+<ErrorCodeTitle code='RUNTIME-001'/>
+
+
+## Reasons
+
+The remoteEntryGlobalName should be got in globalThis if the remote entry file is loaded normally.
+However, it is not available during this load. There are three possible reasons for this error:
+
+1. The remoteEntryUrl is not right .
+2. The remoteEntry file does not mount the container correctly .
```

Comment:
This description of the user may not be able to understand, I think it can be changed to remote entry does not have a normal registration on the window, can tell how to see